src/key-store.js

```diff
@@ -74,8 +74,17 @@ const verifyMessage = async (signature, publicKey, data) => {
   return res
 }
 
+const defaultPath = './keystore'
+
+/**
+ * Creates an instance of KeyStore.
+ * @param {Object} options Various options to use when instantiating KeyStore.
+ * @param {Object} options.storage An instance of a storage class. Can be one of ComposedStorage, IPFSBlockStorage, LevelStorage, etc. Defaults to ComposedStorage.
+ * @param {string} options.path The path to a valid storage. Defaults to ./keystore.
+ * @return {KeyStore} An instance of KeyStore.
+ */
 const KeyStore = async ({ storage, path } = {}) => {
-  storage = storage || await ComposedStorage(await LevelStorage({ path: path || './keystore' }), await LRUStorage({ size: 1000 }))
+  storage = storage || await ComposedStorage(await LevelStorage({ path: path || defaultPath }), await LRUStorage({ size: 1000 }))
```

Comment:
This should be the other way around: we first store the value in LRU and then on disk (level).